apps/docs/content/components/forms/TextArea.mdx

```diff
@@ -1,49 +1,119 @@
 ---
 title: TextArea
-description: TBD
+description: A TextArea serves as a multi-line, plain-text editing interface, ideal for accommodating substantial amounts of free-form text input from users.
 category: "forms"
 links:
     source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/inputs/src/TextArea.tsx
-status: WIP
 ---
 
+<Example src="inputs/docs/textArea/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+<PropTable component="TextArea" />
+
+## Anatomy
 
-## Guidelines
+<FeatureFlag flag="next">
+    ## Guidelines
 
-TODO: If we have some guidelines about this component's usage
+    TODO: If we have some guidelines about this component&apos;s usage
 
-### Accessibility ?
+    ### Accessibility?
 
-TODO: If we have some guidelines about this component and accessibility
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
-## Anatomy
+<FeatureFlag flag="rc">
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-### Concepts
+    ### Concepts
 
-TODO: links to related concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
 ### Composed Components
 
-TODO: links to related components (most likely all component which are used by the slots)
+A `TextArea` uses the following components.
+
+- [Label](Label)
+- [Helper Message](HelperMessage)
+- [Error Message](ErrorMessage)
 
 ## Examples
 
-## Advanced customization
+### Disabled
+A TextArea in a disabled state shows that an input field exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+
+<Example src="inputs/docs/textArea/disabled" />
+
+### ReadOnly
+The `isReadOnly` prop makes the TextArea's text content immutable. Unlike `isDisabled`, the TextArea remains focusable and the contents can still be copied. See [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information.
+
+<Example src="inputs/docs/textArea/readOnly" />
+
+### Error
+A TextArea can be displayed in an error state to indicate that the user input is invalid.
+
+<Example src="inputs/docs/textArea/error" />
+
+### Sizes
+TextAreas have multiple sizes to choose from.
```

Comment:
yes it's still multiple sizes. Someone, maybe you, previously mentioned that we want it to be general in case we add more sizes.
I could state that the size only affects the font-size, but if this ever changes, then we'd need to remember to update it.

Reply:
you are right